### PR TITLE
Updating and improving the Readme layout

### PR DESCRIPTION
**Pull request by jtyost2**
Closes #35 

Focused on improving the content and layout of the Readme, providing clearer setup and installation instructions.

### CODE REVIEW

**Comment by jtyost2** [comment]
@loadsys/developers @loadsys/owners 

Feedback on the Updated Readme.

File: README.md [hunk 1]
```diff
@@ -90,7 +102,438 @@ $this->request->data = array(
...
+		'first_name',
+		'last_name'
+	);
+}
+```
+
+This Serializer will throw a `SerializerMissingRequiredException` if the data passed
+to the Serializer does not include all of the required properties.
+
+All other fields (in this exampled `created` and `modified`) will be suppressed
```
Comment (beporter) [inline README.md]: `exampled` :arrow_right: `example

**Comment by beporter** [comment]
That was the only thing that I saw. The rest reads really well, but I know how they work so I might have been unconsciously filling in any blanks from my own experience. I'd love to get @kriskd's thoughts on whether this is an improvement.

**Comment by ricog** [comment]
I like that a lot better. Good work organizing it @jtyost2

**Pull request merged**